opentelemetry-etw-logs/src/logs/exporter_options.rs

```diff
@@ -0,0 +1,262 @@
+use opentelemetry::otel_warn;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+/// TODO: Add documentation
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum EventMapping {
+    /// TODO: Add documentation
+    HashMap(HashMap<Cow<'static, str>, String>),
+}
+
+/// TODO: Add documentation
+#[derive(Debug)]
+pub struct ExporterOptions {
+    provider_name: Cow<'static, str>,
+    event_mapping: Option<EventMapping>,
+    on_missing_key_use_value: bool,
+    default_event_name: Cow<'static, str>,
+}
+
+impl ExporterOptions {
+    /// TODO: Add documentation
+    pub fn builder(provider_name: Cow<'static, str>) -> ExporterOptionsBuilder {
```

Comment:
```suggestion
    pub fn builder(provider_name: impl Into<Cow<'static, str>>) -> ExporterOptionsBuilder {
```

^^ allows passing strings directly